resource-sharing-partners/src/main/java/no/sikt/rsp/PartnerConverter.java

```diff
@@ -32,17 +41,18 @@ public class PartnerConverter {
     private static final String BIBSYS = "bibsys";
     private static final String ALMA = "alma";
     public static final String OTHER = "OTHER";
+    public static final int RESENDING_OVERDUE_MESSAGE_INTERVAL = 7;
 
     @JacocoGenerated
     public PartnerConverter() {
 
     }
 
-    public static List<Partner> convertBasebibliotekToPartners(BaseBibliotek baseBibliotek) {
+    public static List<Partner> convertBasebibliotekToPartners(String illServer, BaseBibliotek baseBibliotek) {
```

Comment:
Hvis det blir mye stress å sende parametre ned 100 kall så er det bare å endre metodene til non-static for min del og heller ha illServer i konstruktøren.